feat: add `web-ext` for development and pack (#28)
antfu/vitesse-webext: README.md

pnpm dev
```

Then **load extension in browser with the `extension/` folder**.

> For firefox extension developers, just load `extension/` folder to browser, contentScript hmr not work as Chromium based browser.
> 
> As alternative, you can run firefox with command:

Reviewer: ```suggestion
For Firefox developers, you can run the following command instead:
```

pnpm dev
```

Then **load extension in browser with the `extension/` folder**.

For firefox extension developers, just load `extension/` folder to browser,

contentScript hmr not work as Chromium based browser.

> As alternative, you can run firefox with command:

Reviewer 2: These should be one line.

```bash

> pnpm run start:firefox

Reviewer 2: just `pnpm start:firefox`

pnpm build
```

And then pack files under `extension`,

> you can upload `extension.crx` or `extension.xpi` to appropriate extension store.

Reviewer 2: in one line, plz

Author: fixed
